var/spack/repos/builtin/packages/py-neurodamus/package.py

```diff
@@ -39,6 +28,8 @@ class PyNeurodamus(PythonPackage):
     depends_on('py-h5py',          type='run')
     depends_on('py-numpy',         type='run')
     depends_on('py-docopt',        type='run')
+    depends_on('py-scipy',         type='run', when='@2.5.3:')
```

Comment:
Is this now used with py-neurodamus?

Reply:
Used only for local to global positions translation (Baudouin work), and therefore is not even a mandatory dependency in `setup.py`